packages/core/src/amazonqFeatureDev/controllers/chat/messenger/messenger.ts

```diff
@@ -24,8 +24,23 @@ import { FeatureAuthState } from '../../../../codewhisperer'
 import { CodeReference } from '../../../../codewhispererChat/view/connector/connector'
 import { i18n } from '../../../../shared/i18n-helper'
 export class Messenger {
+    /**
+     * Creates an instance of Messenger.
```

Comment:
Were these docstrings generated by AI? Most of these are just restating the function name. That is counterproductive, it's actually worse to have extra text to read which merely duplicates the function name, compared to having no text at all.

Meaningful comments which give insight and hints, are *very* helpful. Meaningless comments are actively harmful.

Reply:
Sorry @justinmk3, this is on draft, as you can see. Isn't on my intent to publish this now.